modules/k8s-nmstate-troubleshooting-dns-disconnected-env-resolv.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * networking/k8s_nmstate/k8s-nmstate-troubleshooting-node-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="k8s-nmstate-troubleshooting-dns-disconnected-env-resolv.adoc_{context}"]
+= Creating a custom DNS host name to resolve DNS connectivity issues
+
+In a disconnected environment where a host server cannot be reached, you can resolve DNS connectivity issues by specifying a custom DNS host name in the `NMState` custom resource definition (CRD). The Kubernetes NMState Operator adds the custom DNS host name to the `/etc/resolv.conf` file. If your cluster gets disconnected, the Operator reads the `/etc/resolv.conf` and uses the custom DNS host name to resolve the DNS connectivity issue.
```

Comment:
I would write

>In a disconnected environment where **external DNS server** cannot be reached, you can resolve **operator health probe issues** by specifying a custom DNS host name in the `NMState` custom resource definition (CRD).

The remaining of the sentence, let's skip it because it's not correct -- I mean, it does not describe the feature we implemented

Reply:
:+1: . I mentioned "Kubernetes NMState Operator" or do we want to be that specific?